src/engraving/libmscore/chordrest.cpp

```diff
@@ -639,17 +659,18 @@ Element* ChordRest::drop(EditData& data)
     }
     break;
 
-    case ElementType::HAIRPIN:
-    {
-        Hairpin* hairpin = toHairpin(e);
-        hairpin->setTick(tick());
-        hairpin->setTrack(track());
-        hairpin->setTrack2(track());
```

Comment:
basically this behaviour makes sense for ALL spanners, no need for it to be a special case for hairpins